chainer_setup_build.py

```diff
@@ -268,3 +328,164 @@ def get_ext_modules():
 
     check_extensions(extensions)
     return extensions
+
+
+def _nvcc_gencode_options():
```

Comment:
This change has not been needed anymore because this PR now uses hardcoded whitelist to generate the values of `--generate-code` option after work with @okuta .

https://github.com/pfnet/chainer/pull/2581/files#diff-f1eae8f2ca7ae61d3658d373d913cea8R362